### DISCUSSION TITLE
wiki/imc/serverjoin

### DISCUSSION THREAD

**Discussion by giscus** [SMGoro/issues-json-generator#10] [General]
# wiki/imc/serverjoin

IMC 服务器加入教程  别问我2022了为啥还有人不会加服，我也不知道（）  IMC.RE 服务器支持Java版/基岩版/网页版三端同服游玩，是国内首创Minecraft三端互通的服务器，我们将尽力打造最优秀的互通服务器游戏体验，将对多渠道设备的服务器游玩体验进行优化！  点击查看客户端下载链接   Java版加服教程 我的世界Java版（Minecraft Java Edition）是我的世

https://blog.imc.re/wiki/imc/serverjoin

**Comment by wearty000123** [comment]
MKWYHTGW

**Comment by xiantiao233** [comment]
wow
